posthog/temporal/data_imports/pipelines/pipeline/pipeline.py

```diff
@@ -164,6 +164,8 @@ def run(self):
                         if len(buffer) >= self._chunk_size:
                             py_table = table_from_py_list(buffer)
                             buffer = []
+                        else:
+                            continue
```

Comment:
surprised this hasn't come up before. This is needed because the meta ads source can return a bunch of < chunk size lists